base/_mixins.scss

```diff
@@ -1,50 +1,100 @@
-// @file mixins.scss
+////
+// Oomph Scaffold Mixins
+// @access public
+// @group Oomph Mixins
+// @author Oomph, Inc. UX Engineers
+// @content the @content directive is not used
+// @parameter none
+// @require no 3rd party requirements
+////
 
-// @link http://csswizardry.com/2016/02/mixins-better-for-performance/
 
-// Provide an accesible hover for non-touch devices, turns it into the active state for mobile, and maintain a no-js fallback
-// @requires Modernizr as a JS dependency to get .no-touch classes
-// Adds styles via @content
-// @jhogue
+// Default constants used by these mixins
+// Change below only if you require a different value
+// These are set by Bourbon
+// $em-base: 16px;
+// These are used by our mixins. Uppercase denotes they are global in scope
+$ICON_FONT_FAMILY: 'font-awesome';
+$DEFAULT_MIN_VW: bp(small);
+$DEFAULT_MAX_VW: bp(x-large);
```

Comment:
Not according to Hugo. https://sass-guidelin.es/#global-flag. The `!default` flag doesn't make sense here either, as these variables are not intended to be overwritten later on. 

I was looking at his notes about Constants, but that doesn't seem to apply here. I'll remove the all caps treatment.